platform-tests/src/test/java/org/junit/platform/console/ConsoleLauncherExecutionResultTests.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2015-2018 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.platform.console;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.Test;
+import org.junit.platform.console.options.CommandLineOptions;
+import org.junit.platform.launcher.listeners.TestExecutionSummary;
+
+public class ConsoleLauncherExecutionResultTests {
```

Comment:
Needs class-level Javadoc with a `@since` tag, and the class does not need to be `public`.